docs/docs/storybook-builder/configuration.md

```diff
@@ -0,0 +1,131 @@
+# Storybook Builder >> Configuration ||2
+
+Web Storybook Builder aims at compatibility with standard Storybook features, e.g. configuration of preview (`.storybook/preview.js`), official addons, etc...
```

Comment:
```suggestion
`@web/storybook-builder` aims at compatibility with standard Storybook features, e.g. configuration of preview (`.storybook/preview.js`), official addons, etc.
```

Reply:
I tried to follow the style of the other doc pages and use a product name rather than package name. Do you think it should be the package?

Reply:
for the same reason also used `Web Dev Server` instead of `@web/dev-server`

Reply:
Aha ok. Hm, I do think the package name is more clear. I don't think its clear to everyone that `Web Dev Server` -> `@web/dev-server`. A "Web Dev Server" is pretty generic

Reply:
ok I'll leave the decision up to you then, I do think `@web/dev-server` is more clear though

Reply:
I'm OK to update then, since I had this concern myself about unclear names

would be nice to also make sure we use these names consistently in other parts of the docs